presto-main/src/main/java/com/facebook/presto/server/protocol/Query.java

```diff
@@ -570,9 +502,9 @@ private synchronized URI createNextResultsUri(String scheme, UriInfo uriInfo)
     {
         return uriInfo.getBaseUriBuilder()
                 .scheme(scheme)
-                .replacePath("/v1/statement")
+                .replacePath("/v1/statement/executing")
                 .path(queryId.toString())
-                .path(String.valueOf(resultId.incrementAndGet()))
+                .path(String.valueOf(nextToken))
```

Comment:
Why change this to `nextToken`?  :)

Reply:
This was supposed to happen in a later commit.  Fixed it.